package.json

```diff
@@ -131,7 +131,7 @@
     "ngx-popperjs": "^15.0.4",
     "ngx-skeleton-loader": "~5.0.0",
     "ngx-translate-messageformat-compiler": "~5.0.1",
-    "ngx-uploader": "~11.0.0",
+    "@angular-ex/uploader": "~15.0.0",
```

Comment:
I don't see it's used somewhere
<img width="393" alt="Screenshot 2023-06-22 at 12 11 11" src="https://github.com/truenas/webui/assets/22980553/10c909da-557a-448f-b4bd-b8451b5400c3">

Reply:
You are right.